pkg/webui/console/components/webhook-form/index.js

```diff
@@ -234,62 +232,49 @@ export default class WebhookForm extends Component {
     initialWebhookValue: undefined,
     onReactivate: () => null,
     onReactivateSuccess: () => null,
-    onSubmitFailure: () => null,
     onDeleteFailure: () => null,
     onDeleteSuccess: () => null,
     onDelete: () => null,
     webhookTemplate: undefined,
-    existCheck: () => false,
     healthStatusEnabled: false,
+    error: undefined,
   }
 
   form = React.createRef()
+
   modalResolve = () => null
   modalReject = () => null
 
   constructor(props) {
     super(props)
-    const { initialWebhookValue } = this.props
+    const { initialWebhookValue, error } = this.props
 
     this.state = {
-      error: undefined,
-      displayOverwriteModal: false,
-      existingId: undefined,
       shouldShowCredentialsInput: Boolean(
         initialWebhookValue?.headers?.Authorization?.startsWith('Basic '),
       ),
+      displayOverwriteModal: false,
+      existingId: undefined,
+      error,
+    }
+  }
+
+  @bind
+  handleReplaceModalDecision(mayReplace) {
+    if (mayReplace) {
+      this.modalResolve()
+    } else {
+      this.modalReject()
     }
+    this.setState({ displayOverwriteModal: false })
   }
 
   @bind
-  async handleSubmit(values, { resetForm }) {
-    const { onSubmit, onSubmitSuccess, onSubmitFailure, existCheck, update } = this.props
+  async handleSubmit(values, { setSubmitting, resetForm }) {
+    const { onSubmit } = this.props
     const castedWebhookValues = validationSchema.cast(values)
     const encodedValues = encodeValues(castedWebhookValues)
```

Comment:
You're missing the `existCheck` here, like you have it in the `<WebhookTemplateForm />`.